catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/campaign/info/campaign_info_state.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:catalyst_voices_view_models/catalyst_voices_view_models.dart';
+import 'package:equatable/equatable.dart';
+
+/// The state of the campaign.
+sealed class CampaignInfoState extends Equatable {
+  const CampaignInfoState();
+}
```

Comment:
does it make sense to simplify this state with just single class those properties? instead 
```dart
final bool isLoading;
final CampaignInfo? campaign;
```